code/modules/mob/living/carbon/human/life/handle_regular_hud_updates.dm

```diff
@@ -271,19 +274,21 @@
 		if(eye_blurry)
 			if(!istype(global_hud.blurry,/obj/screen))
 				global_hud.blurry = getFromPool(/obj/screen)
-				global_hud.blurry.screen_loc = "WEST,SOUTH to EAST,NORTH"
-				global_hud.blurry.icon_state = "blurry"
-				global_hud.blurry.layer = 17
-				global_hud.blurry.mouse_opacity = 0
+				var/obj/screen/blurry = global_hud.blurry //make local for sanic speed
+				blurry.screen_loc = "WEST,SOUTH to EAST,NORTH"
+				blurry.icon_state = "blurry"
+				blurry.layer = 17
+				blurry.mouse_opacity = 0
```

Comment:
None of this shit ever runs, as global_hud is a global var (i should have guessed) and it creates all of this on new. at no point should any of this ever change.

This should be replaced with a `throw` so you all could stop hiding the bug causing the global_hud cache to lose its shit. (or remove it and assume global_hud.blurry is valid as it should never be not valid)